code/src/main/java/com/onlinecontacttracing/messaging/MessageServlet.java

```diff
@@ -0,0 +1,97 @@
+package com.onlinecontacttracing.messaging;
+
+import com.onlinecontacttracing.messaging.LocalityResource;
+import com.onlinecontacttracing.storage.CustomizableMessage;
+import com.onlinecontacttracing.storage.PositiveUser;
+import com.onlinecontacttracing.storage.PotentialContact;
+import com.onlinecontacttracing.messaging.filters.CheckMessagesForFlags;
+import java.util.List;
+
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.Collections;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Projection;
+
+@WebServlet("/messaging")
+public class MessageServlet extends HttpServlet {
+  private SystemMessage systemMessage;
+  private LocalityResource localityResource;
+  private CustomizableMessage customizableMessage;
+  private String userMessage;
+  private PositiveUser user;
+  //for backend use
+  private String compiledMessage;
+  //for frontend display
+  private List<String> statusMessage = new ArrayList<String> ();
+
+
+  public MessageServlet(SystemMessage systemMessage, LocalityResource localityResource, CustomizableMessage customizableMessage, PositiveUser user) {
+    this.systemMessage = systemMessage;
+    this.localityResource = localityResource;
+    this.customizableMessage =  customizableMessage;
+    this.userMessage = customizableMessage.getMessage();
```

Comment:
In general, you don't want to do work in the constructor. Even calling get like this. Probably just store the CustomizableMessage and do the getMessage() call below.